modules/gapi/src/backends/common/serialization.cpp

```diff
@@ -188,14 +188,12 @@ IIStream& operator>> (IIStream& is, cv::RMat&) {
     return is;
 }
 
-IOStream& operator<< (IOStream& os, const cv::MediaFrame &) {
-    // Stub
-    GAPI_Assert(false && "cv::MediaFrame serialization is not supported!");
+IOStream& operator<< (IOStream& os, const cv::MediaFrame &frame) {
+    frame.serialize(os);
     return os;
 }
 IIStream& operator>> (IIStream& is, cv::MediaFrame &) {
-    // Stub
-    GAPI_Assert(false && "cv::MediaFrame serialization is not supported!");
+    util::throw_error(std::logic_error("operator>> for MediaFrame should never be called"));
```

Comment:
Here and in RMat text I'd suggest an alternative. If this operator shouldn't be called, then what should be ?

Reply:
Actually, that's not possible at our level

Reply:
Extended the message though

Reply:
It is, just referring to `cv::gapi::deserialize` -- isn't it?

Reply:
Changed the message